docs/rpc_format.md

```diff
@@ -0,0 +1,64 @@
+# Oasis RPC Format
+
+This doc describes the message format expected by services generated using oasis-rs.
+This is the format used by [oasis.js](https://github.com/oasislabs/oasis.js) when invoking methods on service objects loaded from an IDL.
+
+The types are defined in the [oasis-rpc](/oasis-rpc/src/lib.rs) crate and the wire format is [CBOR](https://tools.ietf.org/html/rfc7049).
+You can find an example of the interface definition language (IDL) in the [tests](/tests/idl-gen/res/TestService.json).
+
+Note: If you find that CBOR is not efficient enough, or don't need struct-as-state semantics, you can build your service using `oasis build --wasi`, which will skip generating an interface or method dispatch tree.
```

Comment:
TODO: We should link to a doc/example doing this. If none exists, let's file an issue. Low priority (obviously).

Reply:
https://github.com/oasislabs/tutorials/issues/33